llm-service/Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM docker-private.infra.cloudera.com/cloudera_base/hardened/cloudera-python:3.10
-RUN pip install pdm
+RUN pip install uv
 COPY ./pyproject.toml /app/
-COPY ./pdm.lock /app/
+COPY ./uv.lock /app/
 WORKDIR /app
-RUN pdm install
+RUN uv install
```

Comment:
The other places used `uv pip install -r pyproject.toml`. Why is this different? 🤔